bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/DYZprime/proc_card_mg5.template

```diff
@@ -0,0 +1,9 @@
+import model_v4 $modelfile
+define p = g u c d s u~ c~ d~ s~
+define j = g u c d s u~ c~ d~ s~
+define l+ = e+ mu+ ta+
+define l- = e- mu- ta-
+define vl = ve vm vt
+define vl~ = ve~ vm~ vt~
+generate p p > zp ,  zp > ta- ta+  QCD=0
```

Comment:
Hi Jose,
Alexander and I talked about this last week.

This should be "p p > zp QCD=0, zp > ta+ ta-", QCD=0 should come before the comma to make pure VBF contributions.
Especially when you use ssmzp_nonuniversal_v4 model file, the coupling between quarks and Z' is alive and you'll get gluon induced Z' diagrams instead of the VBF ones.

Reply:
Hello,

I don't really understand your comment. We are supposed to have Drell-Yan type of Zprime production, not VBF...

Reply:
Ah shoot, sorry.
My memory was mixed up with Amandeep's VBF Zprime requests. QCD=0 will actually play 0 role here.
No need to change anything.